test/release/examples/benchmarks/lcals/LCALSConfiguration.chpl

```diff
@@ -15,6 +15,12 @@ module LCALSConfiguration {
   //
   config const num_suite_passes = 1;
 
+  //
+  // Verification (or quick verification with only a few loop samples)
+  //
+  config const verify_checksums = true;
+  config const verify_checksums_abbreviated = false;
```

Comment:
`verify_checksums_abbreviated` seems like an odd name for this since IMO the main thing it controls is how many sample iterations the main code runs.  The need for different checksums is just a by-product of running fewer iterations.

Reply:
I agree it's a little weird, but I was trying to stay consistent with the reference version. I'm happy to change it to something else if you have a different suggestion.

Reply:
I didn't catch that it came from the reference version.  I'm OK with leaving it.